dashboard/src/shared/Auth.ts

```diff
@@ -42,7 +42,9 @@ export class Auth {
       return [];
     }
     return [
-      "base64url.bearer.authorization.k8s.io." + btoa(token).replace(/=*$/g, ""),
+      // Trimming the b64 padding character ("=") as it is not accepted by k8s
+      // https://github.com/kubernetes/apiserver/blob/master/pkg/authentication/request/websocket/protocol.go#L38
```

Comment:
Since code changes in main branches, best to link to the code in the latest tag imo. (much less likely to change).